feature/experimental/isis/otg_tests/isis_hello_interval_and_multiplier_configuration_test/README.md

```diff
@@ -0,0 +1,55 @@
+# RT-2.16: IS-IS hello-interval and multiplier Configuration Test
+
+## Summary
+
+IS-IS hello-interval and multiplier Configuration Test
+
+## Procedure
+
+* Topology: ATE-port1<—> DUT–port1
+    * Configure IS-IS for ATE port-1 and DUT port-1. 
+    * Establish basic adjacency
+* Baseline Configuration:
+    * Set the hello-interval to a standard value (e.g., 10 seconds).
+    * Set the hello-multiplier to its default (typically 3) 
+    * Check that the ST values are reported correctly.
```

Comment:
What does ST stand for?

Reply:
I have updated it. It stands for Streaming Telemetry